proto/cosmos/gov/v1/gov.proto

```diff
@@ -121,14 +121,14 @@ message VotingParams {
 message TallyParams {
   //  Minimum percentage of total stake needed to vote for a result to be
   //  considered valid.
-  string quorum = 1 [(cosmos_proto.scalar) = "cosmos.Dec", (gogoproto.jsontag) = "quorum,omitempty"];
+  string quorum = 1 [(cosmos_proto.scalar) = "cosmos.Dec"];
 
   //  Minimum proportion of Yes votes for proposal to pass. Default value: 0.5.
-  string threshold = 2 [(cosmos_proto.scalar) = "cosmos.Dec", (gogoproto.jsontag) = "threshold,omitempty"];
+  string threshold = 2 [(cosmos_proto.scalar) = "cosmos.Dec"];
 
   //  Minimum value of Veto votes to Total votes ratio for proposal to be
   //  vetoed. Default value: 1/3.
-  string veto_threshold = 3 [(cosmos_proto.scalar) = "cosmos.Dec", (gogoproto.jsontag) = "veto_threshold,omitempty"];
+  string veto_threshold = 3 [(cosmos_proto.scalar) = "cosmos.Dec"];
```

Comment:
Turns out this proto annotation is useless and doesn't change amino encoding. To convince yourself, take a look at `gov.pb.go` file diff: the jsontags didn't change.